cli/util/file_watcher.rs

```diff
@@ -206,19 +260,41 @@ where
     DebouncedReceiver::new_with_sender();
 
   let PrintConfig {
+    banner,
     job_name,
     clear_screen,
   } = print_config;
 
-  let print_after_restart = create_print_after_restart_fn(clear_screen);
+  let restart_mode = Arc::new(Mutex::new(restart_mode));
+  let print_after_restart = create_print_after_restart_fn(banner, clear_screen);
   let watcher_communicator = WatcherCommunicator {
     paths_to_watch_tx: paths_to_watch_tx.clone(),
     changed_paths_rx: changed_paths_rx.resubscribe(),
     restart_tx: restart_tx.clone(),
+    restart_mode: restart_mode.clone(),
+    banner: colors::intense_blue(banner).to_string(),
   };
-  info!("{} {} started.", colors::intense_blue("Watcher"), job_name,);
+  info!("{} {} started.", colors::intense_blue(banner), job_name);
+
+  let changed_paths = Arc::new(Mutex::new(None));
+  let changed_paths_ = changed_paths.clone();
+
+  tokio::task::spawn(async move {
```

Comment:
Use spawn from deno_core? Then use Rc::RefCell above? Actually, does `restart_mode` need to be in a Mutex or can it be in a RefCell?

Reply:
If a blocking loop is run in JS code then the file changes would never be discovered (as the blocking loop would never yield to the Tokio runtime). I can add a comment here